async_downloads/cache.py

```diff
@@ -148,3 +148,22 @@ def cleanup_collection(collection_key):
     # cache.set(
     #     collection_key, [key for key in cache.get(collection_key, []) if cache.get(key) is not None]
     # )
+
+
+def set_error(download_key, error):
+    download = cache.get(download_key)
+    if not download:
+        return
+
+    logger.exception(
+        f"Download failed with error: {error}",
+        download_key,
+        download.get("name"),
+    )
+
+    download["errors"] = error
```

Comment:
```suggestion
    download["errors"] = str(error)
```
Might be worth doing this just in case we are given an exception object, for example.

Reply:
https://github.com/QuickRelease/django-async-downloads/pull/42/commits/04696e40a46e80bb7b083384b176858d6c5ce931